cmd/runmqserver/qmgr.go

```diff
@@ -86,42 +87,28 @@ func configureQueueManager() error {
 		log.Println(err)
 		return err
 	}
-
 	for _, file := range files {
 		if strings.HasSuffix(file.Name(), ".mqsc") {
 			abs := filepath.Join(configDir, file.Name())
-			// #nosec G204
-			cmd := exec.Command("runmqsc")
-			stdin, err := cmd.StdinPipe()
-			if err != nil {
-				log.Println(err)
-				return err
-			}
-			// Open the MQSC file for reading
-			// #nosec G304
-			f, err := os.Open(abs)
-			if err != nil {
-				log.Printf("Error opening %v: %v", abs, err)
-			}
-			// Copy the contents to stdin of the runmqsc process
-			_, err = io.Copy(stdin, f)
-			if err != nil {
-				log.Errorf("Error reading %v: %v", abs, err)
-			}
-			err = f.Close()
-			if err != nil {
-				log.Errorf("Failed to close MQSC file handle: %v", err)
-			}
-			err = stdin.Close()
-			if err != nil {
-				log.Errorf("Failed to close MQSC stdin: %v", err)
-			}
+			// Pipe the mqsc file into runmqsc
+			cmd1 := exec.Command("cat", abs)
```

Comment:
We should try and use golang functions/libraries as much as possible rather than making system calls.